kanidmd/idm/src/be/dbentry.rs

```diff
@@ -36,6 +431,15 @@ impl std::fmt::Debug for DbEntry {
                 }
                 write!(f, "}}")
             }
+            DbEntryVers::V2(dbe_v2) => {
```

Comment:
Kind-of - it's just more about making it so that when we format these in debug lines they are a bit easier to parse than the default structure that rust gives us. It's used as part of the db_scan toolkit :)